proposals/stage-3/CPC_CHARTER/CPC-CHARTER.md

```diff
@@ -0,0 +1,320 @@
+      for OpenJS Foundation set by the Governing Board, as outlined in the Project Progression Process.
+   1. Voting members are responsible for making final decisions on aligning projects,
+      removing or archiving projects, as outlined in Project Progression Process.
+   1. Voting members are responsible for approving the charter
+      (and any updates) for projects which are part of the OpenJS Foundation. As part of this
+      review/approval the voting members will ensure that the updated charter is compatible
+      with the mission of the OpenJS Foundation and reach consensus with the Board on more
+      substantial changes before final approval by CPC. 
+   1. Voting members are responsible for approving funds for budgets delegated 
+      to the project.
+   1. Following and be up to date on Board/OpenJS Foundation initiatives and communicate them to the projects.
+   1. Defining common practices to be implemented across OpenJS Foundation projects, if any.
+   1. Mediating technical conflicts between OpenJS Foundation Projects when attempts to resolve
+      those conflicts within the Project were unsuccessful. 
+   1. Managing the process for creating a yearly budget and approving a yearly budget to be
+      presented to the board.
+   1. The CPC serves as the OpenJS Foundation's primary technical
+      liaison body with external open source projects, consortiums and groups and is
+      also responsible for ensuring that the OpenJS Foundation has appropriate technical
+      participation in standards bodies by finding an encouraging members from
+      individual projects to participate in these bodies.
+   1. Managing the Individual membership program.
+   1. Managing programs to improve inclusivity and diversity.
+   1. Guiding projects into appropriate Foundation paths.
+   1. Shared resources, tooling and funding for projects and Commons.js (Events
+      Collaborator Summit, Travel fund).
+   1. Supporting projects in their enforcement of Code of Conduct
+      violations and escalation.
+   1. Developing a framework for community end-user feedback.
+   1. Managing programs to offer mentorship to external individuals.
+   1. Managing developer outreach programs.
+   1. Supporting project advocacy and recognition programs.
+   1. Identifying, recruiting and engaging prospective projects.
+   1. Voting members are responsible for making a final decision if consensus
+      cannot be reached among members.
+   1. Members of the CPC, or committees consisting of subsets of the CPC
+      membership will be expected to meet on a regular basis to discuss topics
+      such as new project acceptance into the mentorship program, project graduation
+      from the mentorship program, Project issues and conflicts, opportunities
+      for collaboration between Projects, opportunities for the Foundation in the
+      greater JavaScript community, etc.
+  
```

Comment:
I think we can make this paragraph simpler by splitting the responsibilities of voting members to the ones of regular members in two lists.

Reply:
lets do this in a follow up